CMakeLists.txt

```diff
@@ -10,44 +10,120 @@
 #    (5) Build examples
 #
 #-------------------------------------------------------------------------------
-if(MSVC)
-    # To properly support Visual Studio 2015.
-    set(SIMBODY_REQUIRED_CMAKE_VERSION 3.1.3)
-else()
-    set(SIMBODY_REQUIRED_CMAKE_VERSION 2.8.10)
-endif()
-cmake_minimum_required(VERSION ${SIMBODY_REQUIRED_CMAKE_VERSION})
+cmake_minimum_required(VERSION 3.12)
+cmake_policy(SET CMP0048 NEW)
```

Comment:
This policy is whether to clobber (or not) a `VERSION` cache'd variable. NEW behavior is to clobber; simbody CMake didn't set a VERSION variable, so clobbering isn't a concern.